Makefile

```diff
@@ -204,7 +204,7 @@ endif
 # Set the sysroot.
 override WASM_CFLAGS += --sysroot="$(SYSROOT)"
 
-objs = $(patsubst $(CURDIR)/%.c,$(OBJDIR)/%.o,$(1))
+override objs = $(patsubst $(CURDIR)/%.c,$(OBJDIR)/%.o,$(1))
```

Comment:
Can we skip this patch?  Having override everywhere makes all the lines longer and, for me, makes it harder to read.

I guess the existing overrides are there so that variables can be set on the command line and then appended to?   How about we instread remove *all* the overrides, and not try to support variables both set on the command line and appended to?   Or if there are such variables split them into e.g. CFLAGS and EXTRA_CFLAGS.

Reply:
`override` means the thing *can't* be overridden from the command-line. The default is that everything can be overridden from the command-line. The idea is to try to clearly identify which things people can override and which things they can't. I agree that the syntax is ugly, and admit that it's probably overzealous to rigorously defend against rogue command-line arguments though.

Reply:
My preference would be drop them all then.  I'm not worried about someone who accidentally passes `BASICS_OBJS=x` on the command line.  I'll leave it up to you those.  Not a blocker.

Reply:
I've now removed them.